### PR TITLE
Added sum_series()

### PR DESCRIPTION
**Pull request by archwu**
please review:)

### CODE REVIEW

**Comment by nairraghav** [comment]
Just a note: You do not need to commit your changes for your settings.json from vscode since this is only used locally. It does not break anything but something to keep in mind :)

File: students/ArchWu/Session2/series.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    """Return nth lucas number with low time and space complexity: O(n) time complexity and O(1) space complexity."""
+    a, b = 2, 1
+    if n == 0:
+        return 2
+    for _ in range(n - 1):
+        a, b = b, a + b
+    return b
+
+
+def test_fib():
```
Comment (nairraghav) [inline students/ArchWu/Session2/series.py]: Don't forget the sum_series, refer to the Generalizing area: https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/fib_and_lucas.html

**Comment by nairraghav** [comment]
FizzBuzz and GridPrinter look great, solid job! Please add the sum_series for series.

File: students/ArchWu/Session2/series.py (hunk 1 above)
Comment (archwu) [reply students/ArchWu/Session2/series.py]: I will add them shortly!

**Comment by nairraghav** [comment]
Great job Arch, changes look great!

**Pull request merged**